indexer/categories_holder.cpp

```diff
@@ -143,7 +143,7 @@ void ProcessCategory(string const & line, vector<string> & groups, vector<uint32
 // static
 int8_t constexpr CategoriesHolder::kEnglishCode;
 int8_t constexpr CategoriesHolder::kUnsupportedLocaleCode;
-uint8_t constexpr CategoriesHolder::kMaxSupportedLocaleIndex;
+uint64_t constexpr CategoriesHolder::kMaxSupportedLocaleIndex;
```

Comment:
Nono, please, leave this declaration as is. I suggest you to cast this to uint64_t when declaring SmallSet.

Reply:
done